library/ssl_tls.c

```diff
@@ -4890,7 +4992,7 @@ void mbedtls_ssl_reset_checksum( mbedtls_ssl_context *ssl )
 {
 #if defined(MBEDTLS_SSL_PROTO_SSL3) || defined(MBEDTLS_SSL_PROTO_TLS1) || \
     defined(MBEDTLS_SSL_PROTO_TLS1_1)
-     mbedtls_md5_starts_ret( &ssl->handshake->fin_md5  );
+    mbedtls_md5_starts_ret( &ssl->handshake->fin_md5  );
```

Comment:
Why are we not checking the return code here?

Reply:
Hi @sbutcher-arm 
I wrote to you on https://jira.arm.com/browse/IOTSSL-2049 regarding this function.
I need to change the API of `mbedtls_ssl_reset_checksum` to return `int` but this function is not static.

Reply:
`mbedtls_ssl_reset_checksum()` is an internal function and not supposed to be part of the public interface, therefore you can change it.